deployments/multiple-ragapps/docker-compose.yml

```
@@ -54,19 +54,26 @@ services:
       - KEYCLOAK_ADMIN_PASSWORD=admin
     volumes:
       # Mount example realm data
-      - ./keycloak/example/:/opt/keycloak/data/
+      - ${STATE_DIR:-${PWD}/data}/keycloak/example/:/opt/keycloak/data/
     networks:
       - ragapp-network
+    depends_on:
+      - traefik
 
   manager:
     image: ${MANAGER_IMAGE:-ragapp/manager:latest}
     build: ../../src/manager
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ${STATE_DIR:-${PWD}/data}/manager:/app/config
+      # Mount the ragapps data folder to the manager
+      - ${STATE_DIR:-${PWD}/data}/ragapps:/app/data/ragapps
```

how about just mounting the state dir?
```suggestion
       # Mount the state folder to the manager
      - ${STATE_DIR:-${PWD}/data}:/app/state
```
not sure there's a use case for using a different root folder for manager and ragapps state

---

haha yeah! I also wonder about new use cases that using other data, but it seems to me that we won't have it soon, so just mounting the state folder of ragapp is fine (to avoid the risk)